cmd/vclusterctl/cmd/platform/get/get.go

```diff
@@ -19,5 +19,6 @@ func NewGetCmd(globalFlags *flags.GlobalFlags, cfg *config.CLI) *cobra.Command {
 	}
 
 	cmd.AddCommand(newClusterCmd(globalFlags, cfg))
+	cmd.AddCommand(NewSecretCmd(globalFlags, defaults))
```

Comment:
If `NewSecretCmd` is public, shouldn't `newClusterCmd` also be public?

Reply:
no, it should be private too, just copied over from the loftctl code